### ISSUE TITLE
Moderation Features

### ISSUE THREAD

**Issue by PeterMeyer** [acomminos/Plumble-Legacy#105]
Hi there,
Our Mumble Community needs some Moderationfeatures (the same as in the QT Client)
- Mute a Chatter
- Kick a Chatter
- Move a Chatter
- Bann a Chatter
- Register an Chatter

Are there any plans or Intentions to implement the Features?

Best Greetings, from Germany,
Peter

**Comment by acomminos** [comment]
Yes, there are plans to add administration functionality. Thanks for the report.
